packages/overlay/stories/overlay-element.stories.ts

```diff
@@ -174,6 +174,29 @@ click.args = {
     type: 'auto',
 };
 
+export const withSlider = (): TemplateResult => html`
+    <sp-button id="triggerEl" variant="primary">Button popover</sp-button>
+    <sp-overlay trigger="triggerEl@click" placement="bottom">
+        <sp-popover tip>
+            <sp-dialog no-divider class="options-popover-content">
+                <p>Try clicking the slider after popover opens</p>
+                <p>It snouldn't close the popover</p>
```

Comment:
nit: typo

Reply:
```suggestion
                <p>It shouldn't close the popover</p>
```
This is what I get when I copy/paste other people's work 😜 Thanks for catching this!